src/test/java/com/playground/playground/controller/FeaturesHiddenLayersControllerTest.java

```diff
@@ -0,0 +1,109 @@
+package com.playground.playground.controller;
+
+import com.playground.playground.interface_adapter.controller.FeaturesHiddenLayersController;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.stage.Stage;
+import org.junit.jupiter.api.Test;
+import org.testfx.framework.junit5.ApplicationTest;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class FeaturesHiddenLayersControllerTest extends ApplicationTest {
```

Comment:
No javadocs

Reply:
Wasn't provided for other tests, I'll change that upon merge.

Reply:
You should do it within this PR itself, since this is where the tests are being added

Reply:
Done/